codemods/src/use_utils_assert.ts

```diff
@@ -0,0 +1,103 @@
+/**
+ * Changes imports of `assert` to use `@votingworks/utils` instead.
+ *
+ * ```ts
+ * // broken
+ * import assert from 'assert';
+ *
+ * // fixed
+ * import { assert } from '@votingworks/utils';
+ *
+ *
+ * // broken
+ * import { strict as assert } from 'assert';
```

Comment:
The test cases imply to me that this case can't be fixed by the codemod so I'm confused by it being used as an example here, am I reading the tests wrong? That said I think this is the most common way we import from assert (or at least what I usually do) so it seems worth supporting if we can.

Reply:
This example is supported. I specifically allow `strict` and `ok` named imports.